cobra_test.go

```diff
@@ -513,6 +523,11 @@ func TestSubCmdTakesArgs(t *testing.T) {
 func TestCmdOnlyValidArgs(t *testing.T) {
 	result := noRRSetupTest("echo times one two five")
 
+	if result.Error == nil {
+		t.Errorf("Expected an error")
+		t.FailNow()
```

Comment:
The `testing` package provides:
```go
t.Fatal("Expected an error")
```

as a convenience method for wrapping the two calls you're making here.